src/openzaak/components/zaken/api/validators.py

```diff
@@ -280,6 +281,94 @@ def __call__(self, attrs: dict):
                 raise serializers.ValidationError(self.message, code=self.code)
 
 
+class DeelzaakReopenValidator:
```

Comment:
I'm not sure where, but it would be nice if the validation implemented by this validator and `EndStatusDeelZakenValidator` are mentioned somewhere in the API spec